plugins/offboard/offboard.h

```diff
@@ -152,6 +158,13 @@ class Offboard : public PluginBase
      */
     void set_velocity_body(VelocityBodyYawspeed velocity_body_yawspeed);
 
+    /**
+     * @brief Set the actuator control targets for a specific control group.
+     *
```

Comment:
I think this is perhaps this is an advanced concept that we cannot assume users will know about. Perhaps provide a very brief overview and link to appropriate docs: https://dev.px4.io/en/concept/mixing.html#control-groups